tasks/office.py

```diff
@@ -142,6 +135,9 @@ def get_move(self, overrides=None):
             **self.user.cert_kwargs,
         )
         new_mto, success = check_response(resp, "getMove")
+        if success:
+            self.add_stored(MOVE_TASK_ORDER, new_mto)
+        return new_mto
```

Comment:
Should this be indented, or do you want to return the payload regardless of if it was a success or not? I think in the Prime we just return None

Reply:
I don't think I meant for it to be indented, I don't feel that strongly about it though.  I don't think Python has a strict a convention on return values as Go because of the error return type.

Reply:
The reason we might want to indent this is because receiving a `None` from this function would be a clear way to indicate that the endpoint call was not successful. I _think_ in the Prime tasks, we assume that if we get a JSON response, that it is the JSON for a 200 response. 

It shouldn't cause an issue if you're handling the returned values carefully (and I think you are), but I guess I just like returning None as a way to indicate failure. I really don't have that strong of an opinion about this either, so whichever way feels best to you